itest/litd_test_list_on_test.go

```diff
@@ -48,6 +48,10 @@ var allTestCases = []*testCase{
 		name: "test custom channels liquidity",
 		test: testCustomChannelsLiquidityEdgeCases,
 	},
+	{
+		name: "test custom channels htlc success",
```

Comment:
I guess we'll also want to add a test case for timing out the HTLCs, instead of resolving them?

Reply:
Bad test name here. We timeout out HTLCs at the very end of the itest when we mine enough blocks to timeout the HTLCs.

Reply:
So this tests all 4 scenarios we about ( (local, remote) x (success, timeout) ), it does it all in one sweep with Alice's commitment that she force closed. I'll change the name of the test to better reflect what's covered.